srcpkgs/python3-matplotlib/template

```diff
@@ -14,7 +15,7 @@ maintainer="Alessio Sergi <al3hex@gmail.com>"
 license="custom:matplotlib, BSD-3-Clause, MIT"
 homepage="https://matplotlib.org/"
 distfiles="https://github.com/matplotlib/matplotlib/archive/v${version}.tar.gz"
-checksum=5462728ed3be60af21bd8a6b33f5f1632dabdb3c1b3cc279cffb05926a48255c
+checksum=ee72a23bd5310a69a96c8527901d4ce71820bcbee260c04d1ef2069201c9b98a
 
```

Comment:
Define a `pre_build` function here:
```
pre_build() {
	cat <<-EOF > setup.cfg
	[libs]
	system_freetype = true
	system_qhull = true
	EOF
}
```
The build failures occur when trying to build an in-tree copy of `freetype`, but we can just link to the version shipped with Void. While we're at it, we might as well link to `libqhull` that's provided by Void.